### PR TITLE
wordgrinder 0.6.0 (new formula)

### PR DESCRIPTION
**Pull request by benjaminweb**

### CODE REVIEW

**Comment by dunn** [comment]
```
In file included from src/c/arch/unix/x11/glyphcache.c:7:
src/c/arch/unix/x11/x11.h:9:10: fatal error: 'X11/Xlib.h' file not found
#include <X11/Xlib.h>
         ^
```
I'm afraid that formulae that depend on X11 need to be submitted to our [X11 tap](https://github.com/Homebrew/homebrew-x11/).

There's also the issue of notability:
```
==> audit problems
wordgrinder:
 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
```
We want to be confident formulae we accept will continue to be maintained for the forseeable future, and notability is a relatively good indicator.

File: Library/Formula/wordgrinder.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  head "https://github.com/davidgiven/wordgrinder"
+
+  depends_on "lua"
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    bin.install "bin/wordgrinder" => "wordgrinder"
+    bin.install "bin/wordgrinder-static" => "wordgrinder-static"
+    bin.install "bin/wordgrinder-debug" => "wordgrinder-debug"
+    man1.install "wordgrinder.man" => "wordgrinder.1"
```
Comment (bfontaine) [inline Library/Formula/wordgrinder.rb]: Is there no `make install` we can use?

File: Library/Formula/wordgrinder.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  test do
+    system "wordgrinder", "--help"
```
Comment (bfontaine) [inline Library/Formula/wordgrinder.rb]: Can’t we write a test that does something more substantial than e.g. `--version`/`--help`?

**Comment by bfontaine** [comment]
Ping?

File: Library/Formula/wordgrinder.rb (hunk 2 above)
Comment (benjaminweb) [inline Library/Formula/wordgrinder.rb]: Thanks for following up.
I changed my mind on this formula. I don't want to maintain it any more.
Feel free to dismiss it.
Would that make sense for a tool intended mainly for interactive usage?

Comment (bfontaine) [inline Library/Formula/wordgrinder.rb]: > Would that make sense for a tool intended mainly for interactive usage?

Not really. Sometimes these tools provide options for non-interactive use but if it’s not the case `--help` is ok.

**Comment by bfontaine** [comment]
> I changed my mind on this formula. I don't want to maintain it any more.
> Feel free to dismiss it.

Closing this then.

**Pull request closed without merging**